tools/db_bench_tool.cc

```diff
@@ -4610,7 +4630,16 @@ class Benchmark {
         FLAGS_level0_file_num_compaction_trigger;
     options.level0_slowdown_writes_trigger =
         FLAGS_level0_slowdown_writes_trigger;
-    options.compression = FLAGS_compression_type_e;
+    if (!strcasecmp(FLAGS_compression_manager.c_str(), "mixed")) {
+      options.compression = kZSTD;
```

Comment:
Maybe add explanation about why do we need to set kzstd here instead for future developer